src/pages/Scan.vue

```diff
@@ -36,15 +41,15 @@
                     <button id="submitVoucher"
                         ref="submitVoucher"
                         v-on:click="onRecordVoucher"
-                        v-bind:class="[{ spinner: this.spinner }, { validate: this.validate }, { fail: this.fail }]"
+                        v-bind:class="[{ spinner: this.spinner }, { validate: this.validate }, { fail: this.fail }, { queued: this.queued }]"
                         class="cta"
                     ><span class="hidden offscreen">Submit code</span></button>
 
                 </form>
 
             </div>
 
-            <div v-if="this.vouchers.length > 1">
+            <div v-if="this.vouchers.length >= 1 && !this.netMgr.online">
```

Comment:
The queue nees to show only if the voucher list is > 1 I think.

Once online - but not submitted yet, they still need to see it.